workflows/ampliseq.nf

```diff
@@ -117,6 +117,11 @@ if ( params.dada_ref_taxonomy && !params.skip_dada_addspecies && !params.skip_da
     }
 }
 
+// Error message for incompatible combination of --orf_start and --orf_end 
+if ( params.orf_end && ( ( ( params.orf_end + 1 ) - params.orf_start ) % 3 != 0 ) ) {
+    Nextflow.error("Incompatible parameters: The difference of  `--orf_end` and `--orf_start` must be a multiple of 3.")
+}
+
```

Comment:
Thats perfect, but as I said before, `lib/WorkflowAmpliseq.groovy` is where all params checks should be located (some need to be in `workflow/ampliseq.nf`, but not that one).